UnitsNet.Serialization.JsonNet.Tests/UnitsNetJsonConverterTests.cs

```diff
@@ -9,22 +9,22 @@ namespace UnitsNet.Serialization.JsonNet.Tests
 {
     public class UnitsNetJsonConverterTests
     {
-        private readonly JsonSerializerSettings _jsonSerializerSettings;
+        private JsonSerializerSettings JsonSerializerSettings { get; }
```

Comment:
This is probably only a preference thing, but I find private properties peculiar. Usually a property is used to be able to expose it in an interface or to add some implementation to its setters/getters. I don't see how a private getter property adds anything beyond a readonly field. I can appreciate the consistency argument, but in my experience this is a bit against the C# norm.